components/script/dom/performance.rs

```diff
@@ -260,7 +261,7 @@ impl Performance {
             Some(ref timing) => timing.navigation_start_precise(),
             None => self.navigation_start_precise,
         };
-        (time::precise_time_ns() as f64 - nav_start) / 1000000 as f64
+        (time::precise_time_ns() as f64 - nav_start as f64) / 1000000 as f64
```

Comment:
We should be able to write this as `(time::precise_time_ns() - nav_start) as f64 / 1000000.` now.